src/js/script.js

```diff
@@ -0,0 +1,11 @@
+const burgerMenu = document.querySelector('.burger-menu');
+const burgerButton = document.querySelector('.nav__mobile-button');
+const closeButton = document.querySelector('.burger-menu__head button');
+
+burgerButton.addEventListener('click', () => {
+  burgerMenu.classList.add('active');
+})
```

Comment:
Instead of changing the class in an anonymous function callback, you can simplify this (and below) to:

```suggestion
burgerButton.addEventListener('click', () => burgerMenu.classList.add('active')});
```

Reply:
Done